tests/test_parser.py

```diff
@@ -184,6 +184,24 @@ async def test_parsing_docx_type():
     assert results[0]["content"] is not None
 
 
+@pytest.mark.asyncio
+async def test_parsing_xlsx_type():
+    sample = "examples/inputs/sample.xlsx"
+    parser_type = "STATIC_PARSE"
+    results = parse(sample, parser_type)["segments"]
+    assert len(results) >= 1
+    assert results[0]["content"] is not None
+
+
+@pytest.mark.asyncio
+async def test_parsing_pptx_type():
+    sample = "examples/inputs/sample.pptx"
+    parser_type = "STATIC_PARSE"
+    results = parse(sample, parser_type)["segments"]
```

Comment:
Failing on `AUTO` model for .pptx format.

Reply:
Fixed.